app/digests/api.py

```diff
@@ -15,12 +21,29 @@ class DigestViewSet(viewsets.ModelViewSet):
     serializer_class = DigestSerializer
     pagination_class = DigestPagination
 
-    content_type = settings.DIGESTS_CONTENT_TYPE
+    content_type = settings.DIGEST_CONTENT_TYPE
+    list_content_type = settings.DIGESTS_CONTENT_TYPE
 
     def _create_response(self, data: Dict[str, Any]) -> Response:
         return Response(data, content_type=self.content_type)
 
-    def list(self, request, *args, **kwargs):
+    def create(self, request: Request, *args, **kwargs) -> Response:
+        try:
+            schema = get_schema(get_schema_name(request.content_type))
+
+            validate_json(request.data, schema=schema)
+            serializer = self.get_serializer(data=request.data)
+            serializer.is_valid(raise_exception=True)
```

Comment:
`validate_json` and `serializer.is_valid`: how to make their different responsibility clear here?

Reply:
`validate_json` is validating against the externally defined schema, whereas `serializer.is_valid` is then validating the actual table fields as using the rules defined in the model.

I thought it was clear enough, I am open to suggestions 👍

Reply:
writing this as two comments?